sierra_adapter/sierra_items_to_dynamo/src/test/scala/uk/ac/wellcome/platform/sierra_items_to_dynamo/services/DynamoInserterTest.scala

```diff
@@ -1,279 +1,293 @@
-import uk.ac.wellcome.platform.sierra_items_to_dynamo.dynamo._
-import uk.ac.wellcome.storage.type_classes.{
-  IdGetter,
-  VersionGetter,
-  VersionUpdater
-}
 import uk.ac.wellcome.platform.sierra_items_to_dynamo.fixtures.DynamoInserterFixture
+import uk.ac.wellcome.storage.ObjectStore
 import uk.ac.wellcome.storage.dynamo._
-
-import scala.concurrent.Future
+import uk.ac.wellcome.storage.fixtures.LocalVersionedHybridStore
+import uk.ac.wellcome.storage.s3.S3Config
+import uk.ac.wellcome.storage.vhs.{
+  EmptyMetadata,
+  VHSConfig,
+  VersionedHybridStore
+}
 import uk.ac.wellcome.test.utils.ExtendedPatience
 
 import scala.concurrent.ExecutionContext.Implicits.global
 
 class DynamoInserterTest
     extends FunSpec
     with Matchers
-    with DynamoInserterFixture
+    with LocalVersionedHybridStore
     with ScalaFutures
     with MockitoSugar
+    with DynamoInserterFixture
     with ExtendedPatience
     with SierraUtil {
 
-  it("ingests a json item into DynamoDB") {
+  it("inserts an ItemRecord into the VHS") {
     withLocalDynamoDbTable { table =>
-      withDynamoInserter(table) { dynamoInserter =>
-        val record = createSierraItemRecord
-
-        val futureUnit = dynamoInserter.insertIntoDynamo(record)
-
-        whenReady(futureUnit) { _ =>
-          Scanamo.get[SierraItemRecord](dynamoDbClient)(table.name)(
-            'id -> record.id.withoutCheckDigit) shouldBe Some(
-            Right(record.copy(version = 1)))
+      withLocalS3Bucket { bucket =>
+        withDynamoInserter(table, bucket) { dynamoInserter =>
+          val record = createSierraItemRecord
+
+          val futureUnit = dynamoInserter.insertIntoDynamo(record)
+
+          whenReady(futureUnit) { _ =>
+            assertStored[SierraItemRecord](
+              bucket = bucket,
+              table = table,
+              id = record.id.withoutCheckDigit,
+              record = record
+            )
+          }
         }
       }
     }
   }
 
   it("does not overwrite new data with old data") {
     withLocalDynamoDbTable { table =>
-      withDynamoInserter(table) { dynamoInserter =>
-        val newRecord = createSierraItemRecordWith(
-          modifiedDate = newerDate,
-          bibIds = List(createSierraBibNumber)
-        )
-        Scanamo.put(dynamoDbClient)(table.name)(newRecord)
-
-        val oldRecord = newRecord.copy(
-          modifiedDate = olderDate,
-          bibIds = List(createSierraBibNumber)
-        )
-
-        val futureUnit = dynamoInserter.insertIntoDynamo(oldRecord)
-        whenReady(futureUnit) { _ =>
-          Scanamo.get[SierraItemRecord](dynamoDbClient)(table.name)(
-            'id -> newRecord.id.withoutCheckDigit) shouldBe Some(
-            Right(newRecord))
+      withLocalS3Bucket { bucket =>
+        withTypeVHS[SierraItemRecord, EmptyMetadata, Assertion](bucket, table) {
+          versionedHybridStore =>
+            val dynamoInserter = new DynamoInserter(versionedHybridStore)
```

Comment:
It seems like you could use the DynamoInserter fixture a bit everywhere in this test suite

Reply:
I was on the fence about it – we need the VHS in the rest of these tests, and you either create it and pass it into a fixture (which is barely any different from the current code), or pass it out in a 2-tuple, which I’m not a fan of.